index.html

```diff
@@ -69,89 +69,11 @@ <h1 class="section-title">
-                <li class="tags">css</li>
-                <li class="tags">JavaScript</li>
-                <li class="tags">html</li>
-              </ul>
-              <button class="buttons">See Project</button>
-            </div>
-          </li>
-          <li class="work">
-            <img src="images/Project2.png" alt="Project image" />
-            <div>
-              <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
-              <ul class="list-tag">
-                <li class="tags">Ruby on rails</li>
-                <li class="tags">css</li>
-                <li class="tags">JavaScript</li>
-                <li class="tags">html</li>
-              </ul>
-              <button class="buttons">See Project</button>
-            </div>
-          </li>
-          <li class="work">
-            <img src="images/Project3.png" alt="Project image" />
-            <div>
-              <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
-              <ul class="list-tag">
-                <li class="tags">Ruby on rails</li>
-                <li class="tags">css</li>
-                <li class="tags">JavaScript</li>
-                <li class="tags">html</li>
-              </ul>
-              <button class="buttons">See Project</button>
-            </div>
-          </li>
-          <li class="work">
-            <img src="images/Project4.png" alt="Project image" />
-            <div>
-              <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
-              <ul class="list-tag">
-                <li class="tags">Ruby on rails</li>
-                <li class="tags">css</li>
-                <li class="tags">JavaScript</li>
-                <li class="tags">html</li>
-              </ul>
-              <button class="buttons">See Project</button>
-            </div>
-          </li>
-          <li class="work">
-            <img src="images/Project5.png" alt="Project image" />
-            <div>
-              <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
-              <ul class="list-tag">
-                <li class="tags">Ruby on rails</li>
-                <li class="tags">css</li>
-                <li class="tags">JavaScript</li>
-                <li class="tags">html</li>
-              </ul>
-              <button class="buttons">See Project</button>
-            </div>
-          </li>
-          <li class="work">
-            <img src="images/Project6.png" alt="Project image" />
-            <div>
-              <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
-              <ul class="list-tag">
-                <li class="tags">Ruby on rails</li>
-                <li class="tags">css</li>
-                <li class="tags">JavaScript</li>
-                <li class="tags">html</li>
-              </ul>
-              <button class="buttons">See Project</button>
-            </div>
-          </li>
+        <ul id="list-work" class="list-work">
```

Comment:
- [ ] kindly add a closing button in your mobile design , currently once you open a project you cannot close it (in mobile only) , please fix it according to the Figma design 🙏 

![image](https://user-images.githubusercontent.com/89282221/201984469-73d16e4c-4ea2-4258-bea4-e2564a939ce0.png)

_**figma design**_

![image](https://user-images.githubusercontent.com/89282221/201984953-87d9a52f-67e4-467f-b5bb-232ee5dea673.png)